--- Makefile.Linux
+++ Makefile.Linux
@@ -7,7 +7,8 @@
 BUILDDIR = ../bin/
-CC = gcc
+CC = arm-linux-gnueabihf-gcc
 
 CFLAGS = -g  -fPIC -O2 -fno-strict-aliasing -c -pthread -DWITHSPLICE -D_GNU_SOURCE -DGETHOSTBYNAME_R -D_THREAD_SAFE -D_REENTRANT -DNOODBC -DWITH_STD_MALLOC -DFD_SETSIZE=4096 -DWITH_POLL -DWITH_NETFILTER
 COUT = -o 
 LN = $(CC)
 DCFLAGS = 
-LDFLAGS = -fPIC -O2 -fno-strict-aliasing -pthread
+LDFLAGS = -fPIC -O2 -fno-strict-aliasing -pthread -static -static-libgcc -static-libstdc++
 DLFLAGS = -shared
 DLSUFFICS = .ld.so
 # -lpthreads may be reuqired on some platforms instead of -pthreads
